Documentation/gettingstarted/cilium_install_docker.rst

```diff
@@ -0,0 +1,94 @@
+.. _install_cilium_docker:
+
+Cilium, Kubernetes and docker
```

Comment:
docker --> Docker (also make this capitalization change elsewhere).

Reply:
Done